pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html

```diff
@@ -117,9 +117,8 @@
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">The LMSproject team</p>
             <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
                 For additional updates, follow us on
-                <a href='${social.facebook}'    target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Facebook</a>,
-                <a href='${social.instagram}'   target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Instagram</a> or
```

Comment:
Do we have requirement to remove instagram ?

Reply:
I can put this back. I was looking at the docs attached to the ticket and one of the differences was the instagram was removed and twitter and facebook were swapped. But that may have been unintentional so I will check with Whitney on this as well.